images/capi/ansible/roles/node/tasks/main.yml

```
@@ -133,3 +133,7 @@
     src: usr/local/bin/etcd-network-tuning.sh
     dest: "{{ external_binary_path }}/etcd-network-tuning.sh"
     mode: "0755"
+
+- name: Set default HOME_MODE in login.defs (Flatcar)
+  shell: sed -ri "/^#?HOME_MODE\>/s:.*:HOME_MODE 0700:" /etc/login.defs
```

to make it slightly more (could drop the `\>` thing as well
```suggestion
  shell: sed -ri "s/^#?HOME_MODE\>.*/HOME_MODE 0700/" /etc/login.defs
```